client/components/OperationList.js

```diff
@@ -159,6 +166,13 @@ class OperationComponent extends React.Component {
                     <span className="glyphicon glyphicon-file" aria-hidden="true"></span>
                 </a> {label}
             </span>;
+        } else if (op.attachments && op.attachments.url !== null) {
+            maybeAttachment = <span>
+                <a href={op.attachments.url} target="_blank">
+                    <span className="glyphicon glyphicon-link"></span>
```

Comment:
Is it on purpose that you've chosen `glyphicon-link` here in the expanded view, and `glyphicon-file` in the simple view above?